wasm2c/wasm-rt-exceptions.h

```diff
@@ -70,6 +70,69 @@ uint32_t wasm_rt_exception_size(void);
  */
 void* wasm_rt_exception(void);
 
+/**
+ * The maximum size of an exception.
+ */
+#define WASM_EXN_MAX_SIZE 256
```

Comment:
In practice I think this will normally just be a 4-byte pointer.  So maybe 32 is enough for now?

Reply:
256 is what we use today, so we just used that. we would prefer not to break compatibility with existing wasm2c exception handling users (assuming we have some).